src/customFunctions/customFetch.ts

```diff
@@ -0,0 +1,18 @@
+import { bearerTokenAuthorization } from "./../configs/bearerTokenAuthorization";
+
+export const customFetch = async (
+  input: RequestInfo,
+  init: RequestInit = {}
+) => {
+  init.headers = bearerTokenAuthorization.headers;
```

Comment:
teď sice přidáváš headery do každého requestu, ale uživatel této funkce nemá možnost přebít authorization header, kterou tam nsatvuješ ... a dokonce nemá ani možnost nastavit vlastní headers, protože mu je vždycky přemažeš